style.css

```diff
@@ -0,0 +1,727 @@
+  width: 369px;
+  height: 435px;
+}
+
+.context-projects-3 span {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  margin-left: 12px;
+}
+
+.context-projects-3 p {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  margin-left: 15px;
+}
+
+.context-projects-3 ul {
+  padding: 0;
+}
+
+.context-projects-3 ul li {
+  background: rgba(255, 255, 255, 0.24);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 25px;
+  color: white;
+  border: none;
+  height: 25px;
+  margin-bottom: 22px;
+  float: left;
+  list-style: none;
+  text-decoration: none;
+  margin-left: 12px;
+  padding: 12px 10px;
+}
+
+.context-projects-3 ul li:hover {
+  background-color: #f55800;
+  color: white;
+}
+
+.section-about {
+  padding-top: 200px;
+  background: #262626;
+  height: auto;
+  display: grid;
+  grid-template-columns: 1fr;
+  align-content: center;
+  justify-content: center;
+}
+
+.about-text {
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+}
+
+.about-text h2 {
+  grid-column: 1/2;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 72px;
+  line-height: 81px;
+  color: #f55800;
+  margin-bottom: 24px;
+  margin-left: 142px;
+}
+
+.about-text p {
+  grid-column: 1/2;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 32px;
+  color: #f4f5f7;
+  margin-bottom: 24px;
+  margin-left: 142px;
+}
+
+.about-background {
+  grid-column: 2/3;
+  background-image: url("./img/AboutGroup.png");
+  background-repeat: no-repeat;
+  height: 357px;
+}
+
+.about-button {
+  grid-column: 1/2;
+  width: 161px;
+  height: 48px;
+  background-color: #ff6b00;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  border: none;
+  margin-left: 142px;
+}
+
+.about-button:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+  box-sizing: border-box;
+}
+
+.section-about hr {
+  margin-left: 141px;
+}
+
+.about-language > ul li::before {
+  content: "\2022";
+  color: #ff6b00;
+  line-height: 100px;
+  font-size: 32px;
+  margin-left: 12px;
+  position: relative;
+  left: -10px;
+}
+
+.about-language > hr {
+  width: 400px;
+}
+
+.about-language ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-around;
+}
+
+.about-language ul li {
+  content: "\2022";
+  color: white;
+  line-height: 100px;
+  font-size: 32px;
+  margin-left: 12px;
+  left: -10px;
+}
+
+.sublist {
+  display: flex;
+  flex-direction: column;
+}
+
+@media only screen and (max-width: 600px) {
+  .about-button {
+    margin-left: 24px;
+  }
+
+  .about-text p {
+    margin-left: 24px;
+  }
+
+  .about-text h2 {
+    margin-left: 24px;
+  }
+
+  .section-about hr {
+    margin-left: 24px;
+  }
+
+  .about-language ul {
+    flex-direction: column;
+  }
+}
+
+@media only screen and (max-width: 900px) {
```

Comment:
Please use the value for min-width and max-width property in range of 992px - 1100px.